tasks/main.yml

```diff
@@ -10,7 +10,11 @@
   user: name={{kafka_user}} system=yes group={{kafka_group}}
 
 - name: Fetch kafka binary package
-  get_url: dest=/root url="{{apache_mirror}}/kafka/{{kafka_version}}/kafka_2.9.2-{{kafka_version}}.tgz"
+  get_url: dest=/root url="{{apache_mirror}}/kafka/{{kafka_version}}/kafka_2.9.2-{{kafka_version}}.tgz" timeout=30
```

Comment:
That timeout is not the total download timeout, it is just the timeout for any blocking operation like connect. I had to dig through the source to determine that